src/cli/arguments.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (C) 2019 Luxoft Sweden AB
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+//
+// SPDX-License-Identifier: MPL-2.0
+
+#include "cli/arguments.h"
+
+#include <glibmm.h>
+
+namespace UserIdentificationManager::Cli
+{
+    std::optional<Arguments> Arguments::parse(int argc, char *argv[], std::ostream &output)
+    {
+        Arguments arguments;
+        Glib::OptionGroup main_group("main", "Main Options");
+        Glib::OptionContext context;
+
+        {
+            Glib::OptionEntry entry;
+            entry.set_long_name("version");
+            entry.set_description("Print version and exit");
+            main_group.add_entry(entry, arguments.print_version_and_exit);
+        }
+
+        {
+            Glib::OptionEntry entry;
+            entry.set_short_name('i');
+            entry.set_long_name("identified-users");
+            entry.set_description("Print recently identified users");
+            main_group.add_entry(entry, arguments.print_identified_users);
+        }
+
+        {
+            Glib::OptionEntry entry;
+            entry.set_short_name('s');
+            entry.set_long_name("sources");
+            entry.set_description("Print enabled and disabled identification sources");
+            main_group.add_entry(entry, arguments.print_sources);
+        }
+
+        {
+            Glib::OptionEntry entry;
+            entry.set_short_name('m');
+            entry.set_long_name("monitor");
+            entry.set_description("Monitor for users to be identified");
```

Comment:
Consider change to "Monitor users that are being identified", since I assume this flag will print out any users that are identified by the UIM?

Reply:
Hum, that reads more like the users themselves are being monitored in some way to me. But... what is there now is not that good either. Perhaps "monitor" is a bad name. Don't know... will change to your suggestion, I think.

But yes, this flag just makes it so the cli tool listens for the "UserIdentified" signal and prints id and seat on a single line when it is emitted. (That Ctrl-C can be used to quit is printed before it starts listening.)

Reply:
Changed help text to "Monitor user identification events".